skl2onnx/operator_converters/text_vectoriser.py

```diff
@@ -226,8 +226,11 @@ def convert_sklearn_text_vectorizer(scope: Scope, operator: Operator,
             "You may raise an issue at "
             "https://github.com/onnx/sklearn-onnx/issues.")
 
-    stop_words = op.stop_words_ | (
-        set(op.stop_words) if op.stop_words else set())
+    if hasattr(op, "stop_words_"):
```

Comment:
Is it possible to add a unit test to check the converter when it happens?

Reply:
A unit test is added to tests/test_sklearn_tfidf_vectorizer_converter.py, which accept a custom vocabulary as an additional parameter to TfidfVectorizer. Besides that, is there any modfication needed to adopt the unit test?